### ISSUE TITLE
Use TypeScript definition files from tsd instead of NuGet

### ISSUE THREAD

**Issue by alexangas** [Justin-Credible/Ionic-TypeScript-MDHA-Starter#61]
What do you think about changing to the [tsd](http://definitelytyped.org/tsd/) tool that directly uses the DefinitelyTyped repository rather than having a dependency on NuGet and relying on NuGet packages to be updated? The downside is that I don't think there is UI integration for updates (maybe a grunt or gulp task could be used for this if deemed important).

I've made this change in my test project fork of this one (not on GitHub) and can provide a PR if you like.

**Comment by Justin-Credible** [comment]
This would be similar to how it is done in the [sister project](https://github.com/Justin-Credible/Ionic-TypeScript-Starter), which I actually prefer.

In fact I'm finding I actually prefer the sister project in most cases as using the Visual Studio solution version can be more restrictive in some cases.

This work could have some overlap with [#8](https://github.com/Justin-Credible/Ionic-TypeScript-Starter/issues/8) in the sister project. Ideally, I could add a solution/project file to the sister project and just have one repository.

**Comment by Justin-Credible** [comment]
I guess that doesn't really answer your question. Let me try again... :)

Ideally I would like to deprecate this repo and just add Visual Studio support to the sister project, and in doing so, that would take care of this issue.

**Comment by alexangas** [comment]
It seems like that would be the better approach. Looking back in history, this project was started well before Visual Studio 2015 was close to release so the feature set we have now didn't exist. VS2015 is obviously trying hard to close the problems with cross-platform and perhaps it has achieved that? It would be less maintenance for you and probably easier for users if there was one project.

**Comment by alexangas** [comment]
Something else to consider - this repo has about a third of the stars and forks of the other one. So maybe the other should be deprecated and merged into this one?
